src/uct/ib/rc/accel/gga_mlx5.c

```diff
@@ -29,6 +29,46 @@ typedef struct uct_gga_mlx5_iface_config {
     uct_rc_mlx5_iface_common_config_t rc_mlx5_common;
 } uct_gga_mlx5_iface_config_t;
 
+typedef struct uct_gga_mlx5_dma_opaque {
+    uint32_t syndrom;
+    uint32_t reserved;
+    uint32_t scattered_length;
+    uint32_t gathered_length;
+    uint8_t  reserved2[48];
+} UCS_S_PACKED uct_gga_mlx5_dma_opaque_t;
+
+typedef struct uct_gga_mlx5_dma_opaque_pair {
+    struct mlx5_dma_opaque  *buf;
+    struct ibv_mr           *mr;
+} uct_gga_mlx5_dma_opaque_pair_t;
+
+typedef struct uct_gga_mlx5_iface_qp_cleanup_ctx {
+    uct_rc_mlx5_iface_common_qp_cleanup_ctx_t   super;
```

Comment:
indentation is not according to code style - probably need to review clang-format output for this PR